packages/console/src/pages/UserDetails/components/UserConnectors.tsx

```diff
@@ -127,7 +129,8 @@ const UserConnectors = ({ userId, connectors, onDelete }: Props) => {
                     title="admin_console.user_details.connectors.remove"
                     type="plain"
                     onClick={() => {
-                      void handleDelete(id);
+                      const connector = displayConnectors.find((connector) => connector.id === id);
```

Comment:
Looks like no need to loop again to find the connector here.  Just simply declare the connector in the map method. `displayConnectors.map((connetor) => {})`

Reply:
e.g.?
```ts
const hasConnector = displayConnectors.some((connector) => connector.id === id);
```

Reply:
updated.